client/components/data/current-theme/index.js

```diff
@@ -1,65 +1,67 @@
 /**
  * External dependencies
  */
-var React = require( 'react' );
+import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import omit from 'lodash/object/omit';
 
 /**
  * Internal dependencies
  */
-var CurrentThemeStore = require( 'lib/themes/stores/current-theme' ),
-	Actions = require( 'lib/themes/flux-actions' );
+import { fetchCurrentTheme } from 'lib/themes/actions';
+import { getCurrentTheme } from 'lib/themes/selectors';
+
+function getCurrentThemeState( state, { site } ) {
```

Comment:
Totally up to you, but I notice that other data components just go with a uniform `getState` name, since it's clear, in the context of each file, what the function should do.